### ISSUE TITLE
MailerImpl#close Javadoc clarification

### ISSUE THREAD

**Issue by Technolog33** [bbottema/simple-java-mail#476]
Hi!

I've stumbled across this line of javadoc in MailerImpl#close: `This auto-close is only meant for Spring integration, for when Spring wants to close the Mailer bean.`

Is this supposed to be interpreted in the following way?
I use Spring -> always call #close()
I do not use Spring -> never call #close()

The background being that I am not in a Spring environment, and I'm now not completeley sure whether I should call #close() or not. After all, in like 99% of cases you should call #close() when you have an AutoCloseable, but the comment tells me to refrain from it.
It would be nice to make the javadoc _even more_ clear :)

**Comment by bbottema** [comment]
I understand the confusion. I think the comment can be clearer indeed.

You don't need to call that close manually, Spring invokes it when the bean is disposed of (else the Spring app hangs on the JVM not shutting down).

**Comment by Technolog33** [comment]
That's great to know, but what do I do in a _non_ Spring environment?

**Comment by Technolog33** [comment]
I had read the javadoc of shutdownConnectionPool but that just confused me further.. Perhaps rephrase the line "this is only works in combination with the Advanced batch processing module" to "this is only necessary if the Advanced batch processing module is in use" to clear that up as well.

As I said, I _thought_ that I do not have to call close(), but it just seemed so weird, since not having to call close is a rather rare occurrence.

To sum it up, given that I do not use Spring and also do not use the batch module, I do not need to call close.

Thank you so much for your time! I really like the library overall. It's a pleasure to work with it ^^
